code401Challenges/src/main/java/code401Challenges/BinarySearch.java

```diff
@@ -0,0 +1,26 @@
+package code401Challenges;
+
+public class BinarySearch {
+    public int binarySearch(int[] arr, int num){
+        int index = 0;
+        int low = 0;
+        int high = arr.length - 1;
+        while(low <= high){
+            int mid = (low + high) / 2;
+            if(arr[mid] < num){
+                low = mid + 1;
+            } else if(arr[mid] > num){
+                high = mid - 1;
+            } else if(arr[mid] == num) {
+                index = mid;
+                break;
+            }else if(mid != num){
```

Comment:
Either arr[mid] is less than num, is more than num, or is equal to num. Since you have all those possibilities covered I don't think this final `else if` is reachable